pgml-extension/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "pgml"
-version = "2.7.2"
+version = "2.7.3"
```

Comment:
We haven't released 2.7.2 yet, you can take it.

Reply:
2.7.2, we never knew you.